components/raytac/MDBT50Q.stanza

```diff
@@ -0,0 +1,211 @@
+#use-added-syntax(jitx)
+defpackage ocdb/raytac/MDBT50Q :
+  import core
+  import collections
+  import math
+  import jitx
+  import jitx/commands
+  import ocdb/defaults
+
+  import ocdb/land-patterns
+  import ocdb/symbols
+  import ocdb/box-symbol
+  import ocdb/bundles
+  import ocdb/generator-utils
+  import ocdb/generic-components
+
+public pcb-component component (antenna:String) :
+  manufacturer = "Raytac"
+  mpn = switch(antenna):
+    "chip" : "MDBT50Q-512K"
+    "pcb"  : "MDBT50Q-P512K"
+  description = "nRF52833 based module, with chip or PCB antenna"
+
+  pin-properties :
+    [pin:Ref | pads:Int ...    | side:Dir ]
+    [GND    | 1, 2, 15, 33, 55 | Down ]
+    [NC[0]  | 4                | Up   ]
+    [NC[1]  | 31               | Up   ]
+    [NC[2]  | 42               | Up   ]
+    [NC[3]  | 45               | Up   ]
+    [NC[4]  | 49               | Up   ]
+    [NC[5]  | 59               | Up   ]
+    [NC[6]  | 60               | Up   ]
+    [P0_25  | 3                | Left ]
```

Comment:
These can also be named `P0[25]`, `P0[23]`, if you need to do indexing operations on them.